vsix/ProjectTemplates/VC/Windows Universal/StaticLibrary/cppwinrt_StaticLibrary.vstemplate

```diff
@@ -0,0 +1,45 @@
+<VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
+  <TemplateData>
+    <Name>Static Library (C++/WinRT)</Name>
+    <Description>A project for a C++/WinRT Static Library that can be used by a Universal Windows Platform app</Description>
+    <ProjectType>VC</ProjectType>
+    <SortOrder>3000</SortOrder>
```

Comment:
Note that for VS 2019, you'll also want to add an entry for this template into project-metadata.json, so that it shows up on keyword filters.  I would use a sort order value there of maybe 215.5, so it shows up just beneath the runtime component project.  Example PR:
https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/162259

Reply:
Cool, I'll submit that separate.

Reply:
@Scottj1s looks like I don't have permissions to create PRs there. Can I leave this part to you?